var/spack/repos/builtin/packages/py-basemap/package.py

```diff
@@ -11,19 +11,29 @@ class PyBasemap(PythonPackage):
     """The matplotlib basemap toolkit is a library for plotting
     2D data on maps in Python."""
 
+    url = 'https://github.com/matplotlib/basemap/archive/v1.2.0rel.tar.gz'
     homepage = "http://matplotlib.org/basemap/"
 
+
+    version('1.2.1', sha256='3fb30424f18cd4ffd505e30fd9c810ae81b999bb92f950c76553e1abc081faa7')
     version('1.2.0', sha256='bd5bf305918a2eb675939873b735238f9e3dfe6b5c290e37c41e5b082ff3639a')
     version('1.0.7', sha256='e07ec2e0d63b24c9aed25a09fe8aff2598f82a85da8db74190bac81cbf104531')
 
     # Per Github issue #3813, setuptools is required at runtime in order
     # to make mpl_toolkits a namespace package that can span multiple
     # directories (i.e., matplotlib and basemap)
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-numpy', type=('build', 'run'))
-    depends_on('py-matplotlib', type=('build', 'run'))
-    depends_on('py-pyproj@:1.99', type=('build', 'run'))
-    depends_on('py-pyshp', type=('build', 'run'))
+    depends_on('py-numpy@1.2.1:', type=('build', 'run'))
+    # Also does not work with py-matplotlib@3.0.1; but that "bad" version
+    # is thankfully not in py-matplotlib/packag.epy
+    depends_on('py-matplotlib@1.0.0:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-matplotlib@1.0.0:3.0.0,3.0.2:', type=('build', 'run'))
```
Then you don't need the comment